### PR TITLE
Create batch of contacts

### PR DESCRIPTION
**Pull request by PierreAmmeloot**
I have add information to create batch of new contacts.

### CODE REVIEW

File: source/includes/_api_endpoint_contacts.md [hunk 1]
```diff
@@ -321,6 +321,49 @@ owner|ID of a Mautic user to assign this contact to
...
+lastActive|Date/time in UTC; preferablly in the format of Y-m-d H:m:i but if that format fails, the string will be sent through PHP's strtotime then formatted
+owner|ID of a Mautic user to assign this contact to
+
+#### Response
+
+`Expected Response Code: 201`
+
+** Properties **
+
+Same as [Get Contact](#get-contact).
```
Comment (escopecz) [inline source/includes/_api_endpoint_contacts.md]: Is it the same? It returns an array of contacts, doesn't it?

Comment (PierreAmmeloot) [reply source/includes/_api_endpoint_contacts.md]: Yes it return array of contacts. :-)

**Comment by dmitryuk** [comment]
Hello, do you plan to continue working on the PR?

**Comment by npracht** [comment]
@escopecz would you feel confortable applied the changes you requested ?

File: source/includes/_api_endpoint_contacts.md (hunk 1 above)
Comment (tomekkowalczyk) [reply source/includes/_api_endpoint_contacts.md:365]: @PierreAmmeloot could you correct the returned response?

Comment (dennisameling) [reply source/includes/_api_endpoint_contacts.md]: Just applied the requested fix in https://github.com/mautic/developer-documentation/pull/130/commits/199c9612a4cd166c3ac0eb929de8f7da1caac455 so this PR is now ready to go 🚀

**Review by dennisameling — APPROVED** [review]
Thanks @PierreAmmeloot! Great addition to the docs!

**Pull request merged**